python/ray/tests/test_runtime_env.py

```diff
@@ -830,29 +833,12 @@ def f():
 
 @pytest.mark.skipif(
     sys.platform == "win32", reason="runtime_env unsupported on Windows.")
-@pytest.mark.parametrize(
-    "ray_start_cluster", [{
-        "_system_config": {
-            "event_stats_print_interval_ms": 100,
-            "debug_dump_period_milliseconds": 100,
-            "event_stats": True
```

Comment:
This might be necessary to get the num_workers line to show up in debug_state.txt, but I don't quite remember.  If it works without it that's great!

Reply:
it works without it (somehow?)

Reply:
I see, I'm starting to remember and I think it's fine.  event_stats adds some additional stats to debug_state.txt that we don't need here, and I think I just forgot to remove this part